plugin/PlayAssetDelivery/build.gradle

```diff
@@ -20,13 +20,23 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+
+    testOptions {
+        // return default values so we won't run into Method isEmpty in android.text.TextUtils
```

Comment:
nit: add quotes around the error text, i.e. `so we won't run into "Method isEmpty in android.text.TextUtils not mocked" error during unit test`

Reply:
updated